ops-tools/docker/images/app/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:12-alpine
+WORKDIR /app
+RUN npm install -g nodemon
+COPY package.json /app
+RUN npm install
+COPY app config database public resources start server.js ./app/
```

Comment:
I think you shouldn't do this in docker-compose.yml you did volume for sync files between host and container